frontend/src/__tests__/cypress/cypress/pages/notebookServer.ts

```diff
@@ -44,6 +44,30 @@ class NotebookServer {
   findAcceleratorProfileSelect() {
     return cy.findByTestId('accelerator-profile-select');
   }
+
+  findOpenInNewTabButton() {
+    return cy.findByRole('button', { name: 'Open in new tab' });
+  }
+
+  findSuccessAlert() {
+    return cy.findByText('Success', { timeout: 120000 });
+  }
+
+  findMinimalPythonImage() {
+    return cy.get(
+      '[data-testid="radio jupyter-minimal-notebook"], [data-testid="radio s2i-minimal-notebook"]',
+    );
+  }
+
+  findPythonVersionsButton() {
+    return cy.get('button[aria-controls*="expandable-section-content"]').contains('Versions');
+  }
+
+  findPythonVersion20241() {
+    return cy.get(
+      'input[data-id="jupyter-minimal-notebook:2024.1"], input[data-id="s2i-minimal-notebook:2024.1"]',
```

Comment:
I checked the UI and I was able to select the Radio buttons in ODH cluster using the below:

// Selects Minimal Python 2024.2
`cy.get('[data-testid="radio jupyter-minimal-notebook"]').click();`

I think it would be good to select the Notebook using something like this:
```
findNotebookServer(notebook) {
return cy.get(input'[data-testid="radio ${notebook}"]'),
);
```

Reply:
refactored